docs/examples/koa-webhook-bot.js

```diff
@@ -9,7 +9,11 @@ if (token === undefined) {
   throw new Error('BOT_TOKEN must be provided!')
 }
 
-const bot = new Telegraf(token)
+const bot = new Telegraf(token, {
+  telegram: {
+    webhookReply: false
```

Comment:
No-op without `res`

Reply:
dont understand

Reply:
since the `bot` only responds in specific conditions. so maybe disable webhook is a more generic solution.

Reply:
This option does nothing when you're not passing 2nd argument to `bot.handleUpdate`, so revert this to not complicate the code.

Reply:
ok